duckduckgo/iOS
# Feature/privacy practices screen

Reviewer: Mia
Asana: https://app.asana.com/0/414235014887631/457416080701397/f
CC:

**Description**:

Privacy Protection screen.

**Steps to test this PR**:

1. Visit DDG
1. Observe Good Privacy Practices with green header icon with check, and single good entry

1. Visit Facebook.com
1. Observe Mixed Privacy Practices with red header icon with cross, and two bad entries

1. Visit Steampowerd.com
1. Observe Good Privacy Practices with green header icon with check, two good entries and a single bad entry

1. Visit Google.co.uk
1. Observe Mixed Privacy Practices with red header icon with cross, and three bad entries

1. Visit each privacy protection page
1. Observe correct line height on each page 

* [x] Test 6
1. Tap in the TOSDR link area
1. A new tab is launched with the TOSDR website
